app/backend/middleware/graphql/authenticationPgSettings.js

```diff
@@ -0,0 +1,48 @@
+import { isAuthenticated } from "@bcgov-cas/sso-express";
+
+const authenticationPgSettings = (req) => {
+
+  const claimsSettings = {};
```

Comment:
we're going to need a `role` always returned. Probably hardcode a default role in here and we'll inject the authenticated role lower down